src/test/java/org/isf/disease/test/Tests.java

```diff
@@ -183,6 +190,194 @@ public void testIoIsDescriptionPresent() throws Exception {
 		assertThat(result).isTrue();
 	}
 
+	@Test
+	public void testMgrGetDiseaseByCode() throws Exception {
+		String code = _setupTestDisease(false);
```

Comment:
For tests methods naming at the moment we have a bunch of different approaches all over the code and we should come to a common, documented, decision. But of course this would be out of scope for OP-380 (and in general all OP-147 subtasks). This is due to a lack of documentation from the beginning, so I personally apologyze for this.

I thinks that for the moment we should follow a slightly common pattern for now (like test<MethodName>, test<Package><MethodName>, ecc...) so we will be able to easily refactor all of them at once (or in few steps).

For the

Reply:
For test methods naming at the moment, we have a bunch of different approaches all over the code and we should come to a common, documented decision. But of course, this would be out of scope for OP-380 (and in general all OP-147 subtasks). This is due to a lack of documentation from the beginning, so I apologize for this.

I think that for the moment we should follow a slightly common pattern for now (like test<MethodName>, test<Package><MethodName>, etc...) so we will be able to easily refactor all of them at once (or in few steps).

I'm collecting all the suggestion and I think we can discuss in a decision page, where we can evaluate pros and cons, of also rewriting many tests after many PR from @dbmalkovsky are already ready to be merged.

Reply:
Opened Discussion Page → [Test naming convention ](https://openhospital.atlassian.net/wiki/spaces/OHC/pages/1441071132/Test+naming+convention)